modules/test/playwright/tests/object-web/objectFolder.spec.ts

```diff
@@ -353,6 +353,10 @@ test.describe('manage object definitions through view object definitions', () =>
 
 		await viewObjectDefinitionsPage.deleteObjectFolder(objectFolder.name);
 
+		createdEntities.objectFolders.splice(
```

Comment:
Can we use `apiHelpers.data` instead of having those arrays?

https://github.com/liferay/liferay-portal/blob/f0e6c2aca4abe102a7b0af9c67f2748825a21f35/modules/test/playwright/tests/commerce/commerce-cart-content-web/commerceCart.spec.ts#L33

Reply:
Good idea @ccorreagg. I've changed updated the usage.